db/db_iterator_test.cc

```diff
@@ -2182,6 +2182,45 @@ TEST_P(DBIteratorTest, IteratorWithLocalStatistics) {
             (uint64_t)total_bytes);
 }
 
+TEST_P(DBIteratorTest, IteratorWithPerfContext) {
```

Comment:
TODO: one thing I overlooked is that the TEST_P might not be needed as I don't see how internal details `read_callback_` (i.e, the TEST_P is about) affect simple API counts like this so will consider use TEST_F instead.

Reply:
Fixed